data/mainnet/tokens.js

```diff
@@ -45,6 +45,174 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  quickGamma_MATIC_ETH_narrow: {
+    isNew: true,
+    category: VAULT_CATEGORIES_IDS.QUICKSWAP,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    logoUrl: './icons/wmatic_eth.png',
+    apyIconUrls: ['./icons/quick.png', './icons/wmatic.png'],
+    apyTokenSymbols: ['dQUICK', 'WMATIC'],
```

Comment:
Not all the pools earn wMATIC, please check and only add for the pools that get wMATIC

Reply:
https://quickswap.gamma.xyz/dashboard
Via this link, all pools are getting dQuick and wMatic.